src/js/controller/controller.js

```diff
@@ -248,6 +248,16 @@ Object.assign(Controller.prototype, {
                         index: _model.get('item'),
                         item: playlistItem
                     });
+                    const { title, image } = playlistItem;
+                    if ('mediaSession' in navigator && title && image) {
+                        navigator.mediaSession.metadata = new window.MediaMetadata({
+                            title: title,
+                            artist: window.location.hostname,
+                            artwork: [
+                                { src: image, sizes: '384x384' }
```

Comment:
is `sizes` required? If so, where does '384x384' come from?

Reply:
The image could be any size. But the default thumbnail from our platform is always 720x406* (720 width, height may vary). Ex:
http://assets-jpcust.jwpsrv.com/thumbs/sYKZyjxB-720.jpg

1,920
1,280
480

Reply:
@robwalch @hghazzi works fine without having to use the sizes attribute or having to change the width.   

![screenshot_20180419-164555](https://user-images.githubusercontent.com/6787010/39017985-ccbf13a4-43f2-11e8-9cec-b799a3456b81.png)